### PR TITLE
feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by dlandis**
Great course !

### CODE REVIEW

File: src/content/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -1,7 +1,106 @@
...
+              </BreadcrumbItem>
+            </Breadcrumb>
+            <h1 className="landing-page__heading">
+              Design &amp; build with Carbon
+            </h1>
+          </div>
+        </div>
+      </div>
+      <div className="bx--row landing-page__r2">
+        <div className="bx--row landing-page__r2">
```
Comment (kellychurchill) [inline src/content/LandingPage/LandingPage.js]: Duplicate landing-page__r2 class is causing extra padding in your top row.

**Comment by dlandis** [comment]
Thanks and updated!

File: src/content/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -1,7 +1,104 @@
...
+    tabIndex: 0,
+  },
+};
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width landing-page">
+      <div className="bx--row landing-page__banner">
+        <div className="bx--row landing-page__banner">
```
Comment (kellychurchill) [inline src/content/LandingPage/LandingPage.js]: Sorry I missed this during the first code review. Duplicate div tags here are causing extra padding.

Comment (dlandis) [reply src/content/LandingPage/LandingPage.js]: Thank for the review!

**Pull request closed without merging**